rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -157,10 +158,15 @@ void Player::prepare_publishers()
 {
   auto topics = reader_->get_all_topics_and_types();
   for (const auto & topic : topics) {
+    rclcpp::QoS topic_qos{rclcpp::KeepAll()};
```

Comment:
KeepAll of default profile?

Reply:
It's for the QoS History setting. Constructing a new `rclcpp::QoS` profile requires either `KeepAll()` or `KeepLast(n)`. Everything is set to the ros2 default.

Reply:
Don't we have a default value for `KeepLast`? Or do we need to respecify that every time?

Reply:
rosbag2_transport::Rosbag2QoS should be used AFAICT